arcgis-ios-sdk-samples/Edit data/Create and save KML file/CreateAndSaveKML.storyboard

```diff
@@ -0,0 +1,78 @@
+                                <rect key="frame" x="0.0" y="0.0" width="375" height="623"/>
+                                <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="WEi-Qs-afR">
+                                <rect key="frame" x="0.0" y="623" width="375" height="44"/>
+                                <items>
+                                    <barButtonItem title="Reset" id="VbR-tF-dZd" userLabel="Reset">
+                                        <connections>
+                                            <action selector="resetKML" destination="XDb-RM-FgI" id="N7T-PU-Tcn"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="uYO-iU-ujY"/>
+                                    <barButtonItem systemItem="add" id="V8L-U8-tQY">
+                                        <connections>
+                                            <action selector="addFeature" destination="XDb-RM-FgI" id="Xyb-6T-rhz"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem systemItem="done" id="4d8-3n-68D">
+                                        <connections>
+                                            <action selector="completeSketch" destination="XDb-RM-FgI" id="i9k-ux-qU9"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="ZXj-oh-FBG"/>
+                                    <barButtonItem enabled="NO" systemItem="action" id="MuI-Qf-AEu">
+                                        <connections>
+                                            <action selector="saveKMZ:" destination="XDb-RM-FgI" id="U0V-fV-qEc"/>
+                                        </connections>
+                                    </barButtonItem>
+                                </items>
+                            </toolbar>
+                        </subviews>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <constraints>
+                            <constraint firstItem="WEi-Qs-afR" firstAttribute="trailing" secondItem="lFn-eJ-eZ1" secondAttribute="trailing" id="6ii-kc-5Um"/>
+                            <constraint firstItem="WEi-Qs-afR" firstAttribute="leading" secondItem="lFn-eJ-eZ1" secondAttribute="leading" id="8GR-II-JHm"/>
```

Comment:
The leading and trailing edges of the toolbar do not extend into the safe area.

<img width="881" alt="image" src="https://user-images.githubusercontent.com/2257493/79483974-dbbc6d80-7fc7-11ea-9c8e-f2650d33dfff.png">